coding.md

```diff
@@ -0,0 +1,109 @@
+# Ursula Developer Coding Standards
+In the beginning, there were no guidelines. And it was good. But that
+didn't last long. As more and more people added more and more code,
+we realized that we needed a set of coding standards to make sure that
+Ursula at least *attempted* to display some form of consistency.
+
+In addition, newer versions of Ansible will change what format these
+files can be, so we are being proactive with a coding style to prevent
+having to clean up more things when we move to new versions of Ansible.
+
+Thus, these coding standards/guidelines were developed. Note that not
+all of Ursula adheres to these standards just yet. Some older code has
+just not been changed yet. But be clear, all new code *must* adhere to
+these guidelines.
+
+Below are the patterns that we expect Ursula developers to follow.
+
+## General
+* Our chosen file extension for YAML files is `.yml`.
+
+* Make every attempt to keep lines under 80 chars long.
```

Comment:
This happens naturally when complex args are used. I tend to support a 80 char limit as a *general* rule.  Makes tiling/tabs/working side-by-side easier, and reading in GitHub far better.